src/backend/efiling-api/README.md

```diff
@@ -1,5 +1,24 @@
 # efiling-api
 
+## Maven Profiles
+
+### Openshift
+
+Current image build in openshift runs with `openshift` profile by default.
+This profiles mostly enables splunk logging.
```

Comment:
change to either:
`This profile mostly enables splunk logging.`
OR
`These profiles mostly enable splunk logging.`

Reply:
thanks